packages/core/src/components/dropdown/dropdown-controller.ts

```diff
@@ -89,7 +105,7 @@ class DropdownController {
   }
 
   dismiss(dropdown: DropdownInterface) {
-    if (dropdown.isPresent() && dropdown.willDismiss()) {
+    if (dropdown?.isPresent() && dropdown.willDismiss()) {
```

Comment:
Do we have here a type-definition issue here?

```
dismiss(dropdown: DropdownInterface) // required a none null/undefined instance
```

but the `if(dropdown?.isPresent()` expected a null/undefined variable here?